eventhub/Module.md

```diff
@@ -1,540 +1,199 @@
-```ballerina
-    var listResult = managementClient->listEventHubs();
-    if (listResult is error) {
-        log:printError(listResult.message());
-    }
-    if (listResult is stream<azure_eventhub:EventHub>) {
-        _ = listResult.forEach(isolated function (azure_eventhub:EventHub eventHub) {
-                log:printInfo(eventHub.toString());
-            });
-        log:printInfo("Successfully Listed Event Hubs!");
-    }
-```
-Note:
-This operation will return a ballerina error if the operation failed.
-
-### Step 7: Delete a event hub
-You need to specify the event hub name as a parameter to delete an event hub. This is the basic scenario of deleting 
-an event hub named “mytesthub”. 
-```ballerina
-    var deleteResult = managementClient->deleteEventHub("mytesthub");
-    if (deleteResult is error) {
-        log:printError(msg = deleteResult.message());
-    } else {
-        log:printInfo("Successfully Deleted Event Hub!");
-    }
-```
-Note:
-You can specify the event hub path as a parameter of the deleteEventHub method.
-This operation will return a ballerina error if the operation failed.
 
-# Samples:
+## Quick reference
+Code snippets of some frequently used functions: 
 
-1. Sending an event.
+* Send an event
 
 ```ballerina
-import ballerinax/azure_eventhub as eventhub;
-
-public function main() {
-   eventhub:ClientEndpointConfiguration config = {
-       sasKeyName: "<sas_key_name>",
-       sasKey: "<sas_key>",
-       resourceUri: "<resource_uri>"
-   };
-   eventhub:Client eventHubClient = checkpanic new (config);
-   var result = eventHubClient->send("myhub", "eventData");
-}
+   check eventHubClient->send("myhub", "eventData");
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/send_event.bal
 
-2. Sending an event with broker properties and user properties.
+* Send an event with broker properties and user properties.
 
 ```ballerina
-import ballerinax/azure_eventhub as eventhub;
-
-public function main() {
-   eventhub:ClientEndpointConfiguration config = {
-       sasKeyName: "<sas_key_name>",
-       sasKey: "<sas_key>",
-       resourceUri: "<resource_uri>"
-   };
-   eventhub:Client eventHubClient = checkpanic new (config);
    map<string> brokerProps = {"CorrelationId": "32119834", "CorrelationId2": "32119834"};
    map<string> userProps = {Alert: "windy", warning: "true"};
 
-   var result = eventHubClient->send("myhub", "eventData", userProps, brokerProps);
-}
+   check eventHubClient->send("myhub", "eventData", userProps, brokerProps);
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/send_event_with_broker_and_user_properties.bal
 
-3. Sending an event with broker properties, user properties & partition key.
+* Send an event with broker properties, user properties & partition key.
```

Comment:
broker properties, user properties, and partition key.

Reply:
`Quick reference` section was removed after discussing with the team.